GoogleDataTransportCCTSupport/Tests/Unit/Helpers/GDTCCTEventGenerator.m

```diff
@@ -27,19 +32,106 @@ - (void)deleteGeneratedFilesFromDisk {
 }
 
 - (GDTStoredEvent *)generateStoredEvent:(GDTEventQoS)qosTier {
-  static NSUInteger counter = 0;
   NSString *cachePath = NSTemporaryDirectory();
-  NSString *filePath =
-      [cachePath stringByAppendingPathComponent:[NSString stringWithFormat:@"test-%ld.txt",
-                                                                           (unsigned long)counter]];
-  GDTEvent *event = [[GDTEvent alloc] initWithMappingID:@"1337" target:50];
+  NSString *filePath = [cachePath
+      stringByAppendingPathComponent:[NSString stringWithFormat:@"test-%ld.txt",
+                                                                (unsigned long)gCounter]];
+  GDTEvent *event = [[GDTEvent alloc] initWithMappingID:@"1018" target:kGDTTargetCCT];
   event.clockSnapshot = [GDTClock snapshot];
   event.qosTier = qosTier;
   [[NSFileManager defaultManager] createFileAtPath:filePath contents:[NSData data] attributes:nil];
-  counter++;
+  gCounter++;
   GDTStoredEvent *storedEvent = [event storedEventWithFileURL:[NSURL fileURLWithPath:filePath]];
   [self.allGeneratedEvents addObject:storedEvent];
   return storedEvent;
 }
 
+- (GDTStoredEvent *)generateStoredEvent:(GDTEventQoS)qosTier fileURL:(NSURL *)fileURL {
+  GDTEvent *event = [[GDTEvent alloc] initWithMappingID:@"1018" target:kGDTTargetCCT];
+  event.clockSnapshot = [GDTClock snapshot];
+  event.qosTier = qosTier;
+  gCounter++;
+  GDTStoredEvent *storedEvent = [event storedEventWithFileURL:[NSURL fileURLWithPath:fileURL.path]];
+  [self.allGeneratedEvents addObject:storedEvent];
+  return storedEvent;
+}
+
+- (NSArray<GDTStoredEvent *> *)generateTheFiveConsistentStoredEvents {
+  NSMutableArray<GDTStoredEvent *> *storedEvents = [[NSMutableArray alloc] init];
+  NSBundle *testBundle = [NSBundle bundleForClass:[self class]];
+  {
+    GDTEvent *event = [[GDTEvent alloc] initWithMappingID:@"1018" target:kGDTTargetCCT];
+    event.clockSnapshot = [GDTClock snapshot];
+    [event.clockSnapshot setValue:@(1553536373134) forKeyPath:@"timeMillis"];
+    [event.clockSnapshot setValue:@(-25200) forKeyPath:@"timezoneOffsetSeconds"];
+    [event.clockSnapshot setValue:@(1552576634359451) forKeyPath:@"kernelBootTime"];
+    [event.clockSnapshot setValue:@(961141987648) forKeyPath:@"uptime"];
+    event.qosTier = GDTEventQosDefault;
+    event.customPrioritizationParams = @{@"customParam" : @1337};
+    GDTStoredEvent *storedEvent =
+        [event storedEventWithFileURL:[testBundle URLForResource:@"message-32347456.dat"
```

Comment:
Generated in a one-off project that uses canonical protobuf to write some message protos to disk. This is fine because this is _exactly_ the way the library should see those messages--opaque, without the context of the original .proto. Adopters of this library are responsible for serializing their events to bytes (using proto or whatever), so we just get the bytes.